collections/gamesCollection.mjs

```diff
@@ -10,7 +12,7 @@ class GamesCollection {
     let gameKeys = [],
       subresources = [],
       filters = {};
-    const cb = args.pop();
+    const cb = extractCallback(args);
```

Comment:
When methods explicitly had a callback as a parameter, I could just default it to an empty function. In some cases (like this one), the callback position wasn't pre-defined, so I needed to try to get the last parameter if it was a function, and proceed with the rest of the arguments (if any).